src/openrct2/ride/Vehicle.h

```diff
@@ -294,6 +294,7 @@ struct Vehicle : SpriteBase
     {
         return type == VEHICLE_TYPE_HEAD;
     }
+    void vehicle_update();
```

Comment:
```suggestion
    void Update();
```
There is no need to for a prefix anymore.